Changelog.md

```diff
@@ -1,10 +1,39 @@
 ## Ignition CMake 2.x
 
-### Ignition CMake 2.X.X (2021-08-23)
+### Ignition CMake 2.X.X (2021-XX-XX)
 
-1. Fix problems on IgnOGRE.cmake when version is not found
+### Ignition CMake 2.9.0 (2021-09-01)
+
+1. Fix include directory flags for codecheck
+    * [Pull request #186](https://github.com/ignitionrobotics/ign-cmake/pull/186)
+
+1. Revert FindIgnOgre change
+    * [Pull request #184](https://github.com/ignitionrobotics/ign-cmake/pull/184)
```

Comment:
I believe this is reverting #182? I would just remove both of them from the changelog

Reply:
Good catch. It's not a complete revert though since #182 removes a file. What do you think?

Reply:
It's really reverting #101, which was forward-ported in #182.

Btw, we messed up and squash merged #182 instead of creating a merge commit. That's probably why #101 didn't show up on the changelog and still shows in the branch comparison:

https://github.com/ignitionrobotics/ign-cmake/compare/ign-cmake2...ign-cmake0

(CC @j-rivero )

Reply:
ok, then I would remove #182 and #184 from the changelog and just add an entry for #181 to represent the removed file

Reply:
> ok, then I would remove #182 and #184 from the changelog and just add an entry for #181 to represent the removed file

+1

Reply:
Thanks. Added #181 and removed #182 and #184 in https://github.com/ignitionrobotics/ign-cmake/pull/187/commits/db60cc7f7708436a1852f8abc20266bf7ffb72ae.